### ISSUE TITLE
loss=0.0 when finetuning

### ISSUE THREAD

**Issue by zyandtom** [LLaVA-VL/LLaVA-NeXT#385]
hi, I use my own data and enable lora to finetune `llava-onevision-qwen2-0.5b-si`, found that loss is nearly 0, about 1e-6. Data format has been checked, here is my script
```
META_NAME='test'
OUTPUT_DIR=llava-onevision-qwen2-0.5b-si-$META_NAME
LLM_VERSION="/mnt/ssd2/hf_cache_dir/llava-onevision-qwen2-0.5b-si"
VISION_MODEL_VERSION="/mnt/ssd2/hf_cache_dir/siglip-so400m-patch14-384"
DATA_PATH="/home/yangzeng/data/vlm/open_source_data/maplm_v2/data/val_v2_transformed.json"

PROMPT_VERSION="qwen_1_5"
LLM_VERSION_CLEAN="${LLM_VERSION////}"
VISION_MODEL_VERSION_CLEAN="${VISION_MODEL_VERSION////}"
RUN_NAME="llava-onevision-${VISION_MODEL_VERSION_CLEAN}-${LLM_VERSION_CLEAN}-${META_NAME}"
echo "MID_RUN_NAME: ${RUN_NAME}"

torchrun --nproc_per_node=1 --nnodes=1 \
    llava/train/train_mem.py \
    --deepspeed scripts/zero3.json \
    --model_name_or_path ${LLM_VERSION} \
    --version $PROMPT_VERSION \
    --data_path ${DATA_PATH} \
    --is_customized_data True \
    --image_folder "/home/yangzeng/data/vlm/open_source_data/maplm_v2/data" \
    --lora_enable True \
    --freeze_mm_mlp_adapter True \
    --freeze_mm_vision_resampler True \
    --mm_vision_tower_lr=2e-6 \
    --vision_tower ${VISION_MODEL_VERSION} \
    --mm_projector_type mlp2x_gelu \
    --mm_vision_select_layer -2 \
    --mm_use_im_start_end False \
    --mm_use_im_patch_token False \
    --group_by_modality_length True \
    --image_aspect_ratio anyres_max_9 \
    --image_grid_pinpoints "(1x1),...,(6x6)" \
    --mm_patch_merge_type spatial_unpad \
    --bf16 True \
    --run_name $RUN_NAME \
    --output_dir ${OUTPUT_DIR} \
    --num_train_epochs 1 \
    --per_device_train_batch_size 1 \
    --per_device_eval_batch_size 1 \
    --gradient_accumulation_steps 2 \
    --evaluation_strategy "no" \
    --save_strategy "steps" \
    --save_steps 1000 \
    --save_total_limit 1 \
    --learning_rate 1e-5 \
    --weight_decay 0. \
    --warmup_ratio 0.03 \
    --lr_scheduler_type "cosine" \
    --logging_steps 1 \
    --tf32 True \
    --model_max_length 4096 \
    --gradient_checkpointing True \
    --dataloader_num_workers 4 \
    --lazy_preprocess True \
    --report_to none \
    --torch_compile True \
    --torch_compile_backend "inductor" \
    --dataloader_drop_last True \
    --frames_upbound 32 \
    --attn_implementation sdpa

```

**Comment by zyandtom** [comment]
related to max_length

**Comment by conan1024hao** [comment]
@zyandtom Hi, how did you solve this issue?

**Comment by zyandtom** [comment]
> @zyandtom Hi, how did you solve this issue?

max length is too short so that label will be truncated
